reference/forms/twig_reference.rst

```diff
@@ -217,6 +217,28 @@ array).
 
     <option {% if choice is selectedchoice(value) %} selected="selected"{% endif %} ...>
 
+.. _form-twig-rootform:
+
+rootform
+~~~~~~~~
+
+This test will check if the current ``form`` does not have a parent form view.
+
+.. code-block:: twig
+
+    {# DON'T DO THIS: this code will fail when the form doesn't have a parent
```

Comment:
I'd like to rephrase this sentence... since in no case this code will fail because the `FormView` has a public `parent` property. The problem appears when you have also defined a `parent` form field. The last one will have priority for Twig due the array access ability.

I'm not sure how to explain it in short though.

Reply:
I mean... what you expect for `form.parent` could change depending on whether your form has or not a parent field.